multiplayer/src/components/BetaTag.tsx

```diff
@@ -0,0 +1,10 @@
+export default function Render() {
+    return (
+        <div className="tw-text-sm">
+            <span className="tw-bg-white tw-shadow-sm tw-text-primary-color tw-rounded-md tw-font-bold tw-px-1 tw-pt-[0.05rem] tw-pb-[0.15rem] tw-mr-1">
+                {lf("BETA")}
```

Comment:
Could consider using
```css
    text-transform: uppercase;
```
css rule -- looks like `tw-uppercase`? https://tailwindcss.com/docs/text-transform

Crowdin can get confusing if you have multiple text entries with the same words but different capitalization and no context on why~